Midstone/ComponentFramework/shaders/Explosion.glsl

```diff
@@ -0,0 +1,383 @@
+#version 450
+
+layout(local_size_x = 1, local_size_y = 32, local_size_z = 1) in;
+
+uniform int yDispatch;
+uniform float simSpeed;
+uniform float randSeed;
+
+
+layout(binding = 0) buffer layoutPos
+{
+    vec3 posData[];
+} buf;
+
+layout(binding = 1) buffer layoutVel
+{
+    vec3 vel[];
+} buf2;
+
+
+float rand(float seed) { 
+    return fract(sin(seed) * 43758.5453123); 
```

Comment:
Whats this number?

Reply:
just a large random float used for random number generation